src/Processing.js

```diff
@@ -3235,7 +3235,7 @@
         b = p.lerp(hsb1[2], hsb2[2], amt);
         rgb = p.color.toRGB(h, s, b);
         // ... and for Alpha-range
-        a = p.lerp(a1, a2, amt) * colorModeA;
+        a = p.lerp(a1, a2, amt) * colorModeA + 0.5 | 0;
```

Comment:
good point. It'd be nice to add those parentheses anyway, though, to make the bit wise intent clear to people who end up working with the code as some point in the future.